### PR TITLE
feat(FriendsSince) Add friends icon to popout

### PR DESCRIPTION
**Pull request by trwy7**
Adds the discord friends icon to the Friends Since popout for more visibility. (edited as the last image didn't actually show the feature)
image

### CODE REVIEW

**Comment by Vendicated** [comment]
please show a screenshot

File: src/plugins/friendsSince/index.tsx [hunk 1]
```diff
@@ -47,8 +47,8 @@ export default definePlugin({
                 <Heading variant="eyebrow" className={clydeMoreInfo.title}>
                     Friends Since
                 </Heading>
-
                 <div className={container.memberSinceContainer}>
+                    <svg aria-label="Friends" aria-hidden="false" role="img" xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="none" viewBox="0 0 24 24"><path fill="lightgrey" d="M13 10a4 4 0 1 0 0-8 4 4 0 0 0 0 8Z"></path><path fill="lightgrey" d="M3 5v-.75C3 3.56 3.56 3 4.25 3s1.24.56 1.33 1.25C6.12 8.65 9.46 12 13 12h1a8 8 0 0 1 8 8 2 2 0 0 1-2 2 .21.21 0 0 1-.2-.15 7.65 7.65 0 0 0-1.32-2.3c-.15-.2-.42-.06-.39.17l.25 2c.02.15-.1.28-.25.28H9a2 2 0 0 1-2-2v-2.22c0-1.57-.67-3.05-1.53-4.37A15.85 15.85 0 0 1 3 5Z"></path></svg>
```
Comment (Vendicated) [inline src/plugins/friendsSince/index.tsx:51]:
```suggestion
                    <svg aria-hidden="true" width="18" height="18" viewBox="0 0 24 24"><path fill="lightgrey" d="M13 10a4 4 0 1 0 0-8 4 4 0 0 0 0 8Z"></path><path fill="lightgrey" d="M3 5v-.75C3 3.56 3.56 3 4.25 3s1.24.56 1.33 1.25C6.12 8.65 9.46 12 13 12h1a8 8 0 0 1 8 8 2 2 0 0 1-2 2 .21.21 0 0 1-.2-.15 7.65 7.65 0 0 0-1.32-2.3c-.15-.2-.42-.06-.39.17l.25 2c.02.15-.1.28-.25.28H9a2 2 0 0 1-2-2v-2.22c0-1.57-.67-3.05-1.53-4.37A15.85 15.85 0 0 1 3 5Z"></path></svg>
```

**Comment by trwy7** [comment]
Ready for review

File: src/plugins/friendsSince/index.tsx [hunk 2]
```diff
@@ -47,8 +47,10 @@ export default definePlugin({
                 <Heading variant="eyebrow" className={clydeMoreInfo.title}>
                     Friends Since
                 </Heading>
-
                 <div className={container.memberSinceContainer}>
+                    {Array.from(document.getElementsByClassName("discordIcon_feeef1")).length === 1 && (
```
Comment (Vendicated) [inline src/plugins/friendsSince/index.tsx:51]: do not use raw dom

**Comment by Vendicated** [comment]
by do not use raw dom i meant do not use document.querySelector at all. it's very hacky and not the ideal way to do it

**Comment by Vendicated** [comment]
thank you for the contribution!

this has a loot of issues such as
- dom manipulation instead of using discord module
- hardcoded colour instead of using discord css variable
- wrong size (should be 16px, not 18px)
- bad formatting

as such, i have fixed these issues myself and commited it (with you as co-author) via https://github.com/Vendicated/Vencord/commit/38ffdd7d9457858c6a79a5705f4c7746047f7561

**Pull request closed without merging**